### PR TITLE
Fixing #350

### LINKED ISSUE #350: [BUG] Unit test syntax
**Describe the bug**
Following #339 , syntax changed for reading in IFC files. Some unit tests were corrected, I fear that I have missed some.

**Expected behavior**
All unit tests should compile.

**CMake configuration**
If applicable, list any special configurations you have selected in CMake.
- EARLYBINDING_WITH_IFC4x1: selected

### PR DESCRIPTION
**Pull request by SamuilsRulovs**
Fixes #350

### CODE REVIEW

File: UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp [hunk 1]
```diff
@@ -110,14 +109,14 @@ TEST_F(BasinTessellation, PlaneSurfaceViews)
...
-	/*
+	
 	buw::storeImage(testPath("basin-tessellation_front.png").string(), image_front);
 	buw::storeImage(testPath("basin-tessellation_top.png").string(), image_top);
 	buw::storeImage(testPath("basin-tessellation_bottom.png").string(), image_bottom);
 	buw::storeImage(testPath("basin-tessellation_left.png").string(), image_left);
 	buw::storeImage(testPath("basin-tessellation_right.png").string(), image_right);
 	buw::storeImage(testPath("basin-tessellation_back.png").string(), image_back);
-	*/
+
```
Comment (pjanck) [inline UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp]: This should stay commented out.

File: UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp [hunk 2]
```diff
@@ -166,7 +165,7 @@ TEST_F(BasinTessellation, VertexViews)
 	buw::Image4b image_right_bottom_back = CaptureImage();
 
 	// uncomment following lines to also save the screen shot
-	/*
+
```
Comment (pjanck) [inline UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp]: This should stay commented out.

File: UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp [hunk 3]
```diff
@@ -175,7 +174,7 @@ TEST_F(BasinTessellation, VertexViews)
 	buw::storeImage(testPath("basin-tessellation_top_right_back.png").string(), image_top_right_back);
 	buw::storeImage(testPath("basin-tessellation_back_left_bottom.png").string(), image_back_left_bottom);
 	buw::storeImage(testPath("basin-tessellation_right_bottom_back.png").string(), image_right_bottom_back);
-	*/
+
```
Comment (pjanck) [inline UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp]: This should stay commented out.

File: UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp [hunk 4]
```diff
@@ -117,14 +116,14 @@ TEST_F(BathCsgSolid, PlaneSurfaceViews)
...
-	/*
+	
 	buw::storeImage(testPath("bath-csg-solid_front.png").string(), image_front);
 	buw::storeImage(testPath("bath-csg-solid_top.png").string(), image_top);
 	buw::storeImage(testPath("bath-csg-solid_bottom.png").string(), image_bottom);
 	buw::storeImage(testPath("bath-csg-solid_left.png").string(), image_left);
 	buw::storeImage(testPath("bath-csg-solid_right.png").string(), image_right);
 	buw::storeImage(testPath("bath-csg-solid_back.png").string(), image_back);
-	*/
+
```
Comment (pjanck) [inline UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp]: Similarly.

File: UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp [hunk 5]
```diff
@@ -173,7 +172,7 @@ TEST_F(BathCsgSolid, VertexViews)
 	buw::Image4b image_right_bottom_back = CaptureImage();
 
 	// uncomment following lines to also save the screen shot
-	/*
+
```
Comment (pjanck) [inline UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp]: Similarly.

File: UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp [hunk 6]
```diff
@@ -182,7 +181,7 @@ TEST_F(BathCsgSolid, VertexViews)
 	buw::storeImage(testPath("bath-csg-solid_top_right_back.png").string(), image_top_right_back);
 	buw::storeImage(testPath("bath-csg-solid_back_left_bottom.png").string(), image_back_left_bottom);
 	buw::storeImage(testPath("bath-csg-solid_right_bottom_back.png").string(), image_right_bottom_back);
-	*/
+
```
Comment (pjanck) [inline UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp]: Similarly.

**Review by pjanck — CHANGES_REQUESTED** [review]

File: UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp (hunk 1 above)
Comment (SamuilsRulovs) [reply UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp]: Resolved: 2ad6aa0

File: UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp (hunk 2 above)
Comment (SamuilsRulovs) [reply UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp]: Resolved: 2ad6aa0

File: UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp (hunk 3 above)
Comment (SamuilsRulovs) [reply UnitTests/Schemas/IFC4X1/basin-tessellation/src/basin-tessellation.cpp]: Resolved: 2ad6aa0

File: UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp (hunk 4 above)
Comment (SamuilsRulovs) [reply UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp]: Resolved: 2ad6aa0

File: UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp (hunk 6 above)
Comment (SamuilsRulovs) [reply UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp]: Resolved: 2ad6aa0

File: UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp (hunk 5 above)
Comment (SamuilsRulovs) [reply UnitTests/Schemas/IFC4X1/bath-csg-solid/src/bath-csg-solid.cpp]: Resolved: 2ad6aa0

**Review by pjanck — APPROVED** [review]
LFTM.

**Pull request merged**